shell/components/SortableTable/index.vue

```diff
@@ -530,9 +532,6 @@ export default {
     manualRefreshLoadingFinished() {
       const res = !!(!this.isLoading && this._didinit && this.rows?.length && !this.isManualRefreshLoading);
 
-      // Always ensure the Refresh button phase aligns with loading state (regardless of if manualRefreshLoadingFinished has changed or not)
-      this.refreshButtonPhase = !res || this.loading ? ASYNC_BUTTON_STATES.WAITING : ASYNC_BUTTON_STATES.ACTION;
-
```

Comment:
This would cause the refresh button to get stuck on page load and refreshButtonPhase expects a phase value not a boolean true or false value.

You'll also notice that we have a manualRefreshLoadingFinished watcher which updates the phase as well so it seems safe to remove.